src/pretix/presale/views/organizer.py

```diff
@@ -231,6 +232,14 @@ def _set_month_year(self):
             except ValueError:
                 self.year = now().year
                 self.month = now().month
+        elif 'date' in self.request.GET:
+            self.tz = self.request.organizer.timezone
```

Comment:
line 225 only sets ``tz``, this sets ``self.tz``, line 229 sets neither. So ``self.tz`` will exist or not exist based on what kind of GET parameters we send? This feels dangerous when we rely on it elsewhere in the code.

Reply:
Not sure where this came from, will set to `tz`. Thanks!

Reply:
Ah, I copied it from `DayCalendarView` where it is used lateron.